rubygem-clamp/rubygem-clamp.spec

```diff
@@ -80,6 +74,7 @@ cp -pa .%{gem_dir}/* \
 %doc %{gem_dir}/doc/%{gem_name}-%{version}
 %{gem_instdir}/README.md
 %{gem_instdir}/CHANGES.md
+%{gem_instdir}/LICENSE
```

Comment:
no need for this line, it's in the main package under %files (and -doc depends on the main package)